<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EraseDataMenu</class>
 <widget class="QMainWindow" name="EraseDataMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="TimeLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>1234567890</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ERASE DATA MENU</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileTypeLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>备注:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE THIS DATA</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileStatusLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EMPTY</string>
    </property>
   </widget>
   <widget class="QLabel" name="transferStatusLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>!!! DATA SET NOT TRANSFERRED	!!!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header>timelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
